index.js

```diff
@@ -234,14 +229,14 @@ const defaultProps = {
   edgeHeight: 20,
   edgeColor: '#FFF',
   edgeBorderWidth: 4,
-  transparency: 0.6,
   showAnimatedLine: true,
   animatedLineColor: '#FFF',
   animatedLineHeight: 2,
   lineAnimationDuration: 1500,
+  backgroundColor: 'rbga(0, 0, 0, 0.6)'
```

Comment:
Could you also update declarations https://github.com/shahnawaz/react-native-barcode-mask/blob/master/index.d.ts#L10 ?

Reply:
It's done.